app/public/css/styles.css

```diff
@@ -1830,6 +1903,45 @@ button[data-balloon] {
   margin-bottom: 64px;
 }
 
+/* REWARDS */
+
+.rewards-wrapper {
+  border-top: 1px solid var(--color-border);
+  margin: 18px 0 0 0;
+}
+
+.rewards-title {
+  margin: 6px 0 0 0;
```

Comment:
perhaps this is personal taste, but changing this to margin-top: ... would be nicer

Reply:
I used this to set the other margins to 0 (it wouldn't be the case otherwise).